src/plugins/comments/test/comments.e2e.js

```diff
@@ -267,6 +283,25 @@ describe('Comments', () => {
       expect(afterSetCellMetaCallback).toHaveBeenCalledWith(1, 1, 'comment', undefined, undefined, undefined);
     });
 
+    it('should not trigger `afterSetCellMeta` callback if `removeCommentAtCell` function is invoked and `beforeSetCellMeta` returned false', () => {
```

Comment:
I'd suggest changing the test to use spyOn on the `setCellMeta` method or check visually if the commend was removed.

I think that the alignment tests which you have written are a good candidate to follow.